java/arcs/core/data/CapabilityNew.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+import arcs.core.util.Time
+
+sealed class CapabilityNew {
+    enum class Comparison { LessStrict, Equivalent, Stricter }
+
+    fun isEquivalent(other: CapabilityNew): Boolean {
+        return compare(other) == Comparison.Equivalent
+    }
+    fun contains(other: CapabilityNew) = isEquivalent(other)
```

Comment:
Do you need all of these or just `LessThan` and construct others?

Reply:
i could, but the constructed others are not very readable, so i think this it preferable.